src/main/java/core/basesyntax/behavior/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.behavior;
+
+public interface AreaCalculator {
+    void areaCalculate();
```

Comment:
```suggestion
    void calculateArea();
```
назва методу починається з дієслова